corehq/apps/app_manager/_design/views/deleted_applications/map.js

```diff
@@ -0,0 +1,8 @@
+function(doc){
+    if(doc.doc_type == 'Application-Deleted' || doc.doc_type == 'RemoteApp-Deleted' || doc.doc_type == 'LinkedApplication-Deleted') {
+        emit([doc.domain, doc.copy_of, doc.version], null);
+        if (doc.is_released && doc.copy_of) {
+            emit(['^ReleasedApplications', doc.domain,  doc.copy_of, doc.version], null);
+        }
+    }
+}
```

Comment:
This view should go in it's own design doc otherwise couch will rebuild ALL views in the app manager design doc. See https://commcare-hq.readthedocs.io/couchdb.html?highlight=couchapps#playing-nice-with-cloudant-couchdb (especially point 6 about putting new views in the [`corehq.couchapps`](https://github.com/dimagi/commcare-hq/blob/master/corehq/couchapps/README.md) app.

Reply:
Also, I don't think you need a new couch view at all. You should be able to use the [`last_modified/by_last_modified`](https://github.com/dimagi/commcare-hq/blob/75557ed0cc39f37d174b4d92286d72819c255a3a/corehq/couchapps/last_modified/views/by_last_modified/map.js#L10-L12) view

Reply:
In that case you could use the `by_domain_doc_type_date/view` view which includes the domain. There are a bunch of utilities in `corehq.apps.domain.dbaccessors` for querying that view like [`iterate_doc_ids_in_domain_by_type`](https://github.com/dimagi/commcare-hq/blob/97410667267b6cbe329503f59ee71751ad194f9c/corehq/apps/domain/dbaccessors.py#L45)